frontend/plugins/payment_ui/src/modules/corporateGateway/golomtbank/configs/containers/List.tsx

```
@@ -0,0 +1,80 @@
+      variables: {
+        limit: 20,
+        ...queryParams,
+      },
+      fetchPolicy: 'network-only',
+    },
+  );
+
+  const [removeMutation] = useMutation(gql(mutations.removeMutation));
+
+  const remove = async (_id: string) => {
+    const confirmed = await confirm(
```

ESLint

[error] Unexpected use of 'confirm'.

(no-restricted-globals)